### ISSUE TITLE
results are not as defined as examples

### ISSUE THREAD

**Issue by ghost** [timxzz/Facial_Expression_Modifier_Backend#1]
I tried this picture
![Bill_Gates_2018](https://user-images.githubusercontent.com/6818285/58024855-25af8180-7ae1-11e9-8176-9ee1e0d11490.jpg)
and got this result
the mouth region shows almost no difference
![Screen Shot 2019-05-20 at 9 25 11 AM](https://user-images.githubusercontent.com/6818285/58024886-36f88e00-7ae1-11e9-9546-ffe17e65a692.png)
also possible to change default crop to not crop hair

**Comment by timxzz** [comment]
Yes, results are depended on the input photos for the current model. What you mentioned are all possible improvements for the project, which were not my main focus at the time. It will be nice if you can come up with better models or more functionalities. Let me know or submit pull request when you do. Thanks!

**Comment by ghost** [comment]
@timxzz i would like to retrain the current model to improve it but not familiar with the current setup, where can I setup training test pair files as right now it is doing inference on the server which I am not familiar with how to setup for training

**Comment by timxzz** [comment]
This is the complete code for the model, which includes both training and testing. And as mentioned in README, the code is based on TF version 1.3.0.
https://github.com/timxzz/GANs/blob/master/Face/Face_StarGAN_v5.ipynb 

And this is for data preprocessing:
https://github.com/timxzz/GANs/tree/master/Face/Face_data

For detail of the algorithm and the training data, please refer for the project report:
http://timx.me/Facial_Expression_Modifier/pdf/Third_Year_Project_Report.pdf

**Comment by ghost** [comment]
@timxzz thanks, the website is not working again

**Comment by Maouriyan** [comment]
> I tried this picture
> ![Bill_Gates_2018](https://user-images.githubusercontent.com/6818285/58024855-25af8180-7ae1-11e9-8176-9ee1e0d11490.jpg)
> and got this result
> the mouth region shows almost no difference
> ![Screen Shot 2019-05-20 at 9 25 11 AM](https://user-images.githubusercontent.com/6818285/58024886-36f88e00-7ae1-11e9-9546-ffe17e65a692.png)
> also possible to change default crop to not crop hair

How did you test ?
